src/main/kotlin/org/malachite/estella/services/JobSeekerService.kt

```diff
@@ -22,10 +23,12 @@ class JobSeekerService(
     fun getJobSeeker(id: Int): JobSeeker = withExceptionThrower { jobSeekerRepository.findByUserId(id).get() } as JobSeeker
 
     fun registerJobSeeker(jobSeeker: JobSeeker): JobSeeker =
-        userService.registerUser(jobSeeker.user).let {
-            jobSeeker.copy(user = it)
-        }.let {
-            jobSeekerRepository.save(it)
+        try {
```

Comment:
Nie lubię try catch ale może być XD

Reply:
Znaczy one są potrzebne, bo tak to byśmy mieli exception handlera na DataValidationIntegrationException, a to niewiele mówi XDDDD